packages/form/addon/components/cf-field/input/action-button.hbs

```diff
@@ -8,6 +8,24 @@
     @validateOnEnter={{this.validateOnEnter}}
     as |isValid validate|
   >
+    {{#if (and this.invalidFields.length @context.showValidation)}}
```

Comment:
As discussed with @czosel: `@context.showValidation` will be replaced by a proper backend configuration property similar to `validateOnEnter`

Reply:
done :+1: